src/test/api/karate/users/perimeters/getCurrentUserWithPerimeters.feature

```diff
@@ -202,7 +206,7 @@ Feature: Get current user with perimeters (endpoint tested : GET /CurrentUserWit
     Then status 200
     And match response.userData.login == 'operator1_fr'
     And assert response.computedPerimeters.length == 2
-    And match response.computedPerimeters contains only [{"process":"process15","state":"state1","rights":"ReceiveAndWrite"}, {"process":"process15","state":"state2","rights":"ReceiveAndWrite"}]
+    And match response.computedPerimeters contains only [{"process":"process15","state":"state1","rights":"ReceiveAndWrite","filteringNotificationAllowed":true}, {"process":"process15","state":"state2","rights":"ReceiveAndWrite","filteringNotificationAllowed":true}]

```

Comment:
add in this test a value false for filteringNotificationAllowed

Reply:
Done.